modules.d/95iscsi/parse-iscsiroot.sh

```diff
@@ -47,13 +47,6 @@ if [ "${root%%:*}" = "iscsi" ]; then
     write_fs_tab /dev/root
 fi
 
-# If it's not empty or iscsi we don't continue
```

Comment:
Is this change (with the check in 137) functionally equivalent or does it change behavior?

Reply:
I will double check again, after clearing my head with all the shellcheck

Reply:
reverted the logic checks

Reply:
opened https://github.com/dracutdevs/dracut/issues/1279